src/Propel/Runtime/Propel.php

```diff
@@ -267,29 +267,36 @@ public static function getLogger()
      * @param string $message The message that will be logged.
      * @param int    $level   The logging level.
      *
-     * @return boolean True if the message was logged successfully or no logger was used.
+     * @return void
```

Comment:
Why don't you want to forward the return?

Reply:
See contract of logging. All void.

Reply:
`\Monolog\Logger::debug` and related methods have returns.

Reply:
Yes, they violate their own interface. But nothing useful is returned anyway.

Reply:
Also propel never uses the return value already, so all fine IMO.

Reply:
Check our generated code like:

`\Orm\Zed\CompanyUserInvitation\Persistence\Base\SpyCompanyUserInvitation::log`

```
    protected function log($msg, $priority = Propel::LOG_INFO)
    {
        return Propel::log(get_class($this) . ': ' . $msg, $priority);
    }
```

All generated base classes have this expectation.

Reply:
True, but as long as we dont add a typehint, returning null would still not break anything.
And those log classes are not used right now, so not even any return type relevance here.
We sure should mention this in the migration guide (changelog) for the upcoming tag. But I think given the expectation around the log interface this is the right call here.

Reply:
We should also fix up the generated code though :+1:

Reply:
PS: also in line with https://github.com/Seldaek/monolog/pull/1362/files So moving forward here is better than stepping backward again.